mod_reforged/hooks/skills/skill.nut

```diff
@@ -31,6 +31,8 @@
 				case "Resistance against ranged weapons":	// This was never followed through for anything other than skeletons
 				case "Resistance against piercing attacks":	// This was never followed through for anything other than skeletons
 				case "Nighttime":			// Already displayed on the stats of the character and in the effects section
+				case "On bad terrain":		// Terrain effects usually have every present effects viewable in the actor tooltip
+				case "Target on bad terrain":	// Terrain effects usually have every present effects viewable in the actor tooltip
```

Comment:
reword it as `usually have their effects viewable` instead of "usually have every present effects viewable"